### ISSUE TITLE
Update aws-sdk to v3 in 'Use AWS:S3 As Storage' Documentation

### ISSUE THREAD

**Issue by ThaumRystra** [veliovgroup/Meteor-Files#874]
In the documentation for using [AWS:S3 As Storage](https://github.com/veliovgroup/Meteor-Files/blob/master/docs/aws-s3-integration.md), aws-sdk is at version 2.240.1, which now causes the following warning:

```
Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
```

**Comment by dr-dimitru** [comment]
@ThaumRystra would you love to send a PR?

**Comment by ThaumRystra** [comment]
I'm too dumb to get it to work right, so I am stuck on AWS 2.240 until I figure it out.

I'll submit a PR if I manage to get it working properly.

**Comment by jacoatvatfree** [comment]
Also note that trying to use AWS SDK v3 will throw a warning that Node 14 is no longer supported from 1 May 2024, so until meteor switches to a later Node version, there will be warnings.

**Comment by dr-dimitru** [comment]
@jacoatvatfree true, would like to submit a PR, at least as WIP, I will help to finalize, and I'm sure community would join too. Perhaps you have a project from where you can copy-paste general changes required for v3?

If you're going to submit a PR let's postfix files with `v2` and `v3` so we can reference S3 integrations in the docs for all of its versions
